src/widgets/interfaces/get-token.ts

```diff
@@ -1,14 +1,14 @@
-export type WidgetScope = 'widgets:users-table:manage';
+export type WidgetScope = 'widgets:users-table:manage' | 'widgets:sso:manage';
 
 export interface GetTokenOptions {
   organizationId: string;
-  userId: string;
+  userId?: string;
```

Comment:
considered doing something fancy here where `userId` is conditionally required depending on what `WidgetScope`s are requested, but it seemed overly complicated.